### PR TITLE
[AC-9459]: Correct typos on the Business Proposition help text 

### PR DESCRIPTION
**Pull request by kenware**
[AC-9459](https://masschallenge.atlassian.net/browse/AC-9459)

How to test:
- Confirm that all typos has been fixed on http://localhost:8181/startup/20/business-proposition/

Sibling PR: https://github.com/masschallenge/accelerate/pull/3229

### CODE REVIEW

**Comment by iankigen** [comment]
lint issues

File: accelerator_abstract/models/base_business_proposition.py [hunk 1]
```diff
@@ -16,16 +16,18 @@
...
+                   'your customers/users through?')
 COMPETITORS_HELP_TEXT = ('Which organizations compete with'
                          ' your current value offering and who might '
                          'do so in the future?')
-PRODUCT_COMPLEMENT_HELP_TEXT = ('Which organizations compete with your '
-                                'current value offering and who might'
-                                ' do so in the future?')
-PRIMARY_ADVANTAGE_HELP_TEXT = ('What are the primary advantages'
-                               ' over competitors')
+PRODUCT_COMPLEMENT_HELP_TEXT = ('Which organizations and/or product'
```
Comment (iankigen) [inline accelerator_abstract/models/base_business_proposition.py:24]: Typo: Which organizations and/or **products** complement your offering in the market? Do you know of and/or anticipate any value chain partners?

**Review by iankigen — COMMENTED** [review]
Otherwise, looks good

**Review by AmosWels — APPROVED** [review]
QA and changes look good to me. MIgration runs succesfully

**Pull request merged**